megamek/src/megamek/common/AmmoType.java

```diff
@@ -5740,7 +5740,7 @@ private static AmmoType createISLightMGAmmo() {
         .setIntroLevel(false)
         .setUnofficial(false)
         .setTechRating(RATING_B)
-        .setAvailability(RATING_X, RATING_X, RATING_C, RATING_B)
+        .setAvailability(RATING_X, RATING_C, RATING_C, RATING_B)
```

Comment:
In my previous answer I was giving a general answer about weapon and ammo tech advancement data and didn't look closely at the weapon availability codes and it turns out that there's a error in `ISLightMG`. The LMG did not exist in the SW era and the availability is XXCB.
![Screenshot from 2019-06-14 16-33-39](https://user-images.githubusercontent.com/16927464/59539438-df242b80-8ec2-11e9-9346-ad938a332cd0.png)